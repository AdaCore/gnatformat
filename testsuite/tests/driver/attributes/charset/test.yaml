description: |
  Test the Charset attribute but using a source that contains UTF-8 characters.
driver: gnatformat
args: ["-P", "test/test.gpr", "--pipe"]
