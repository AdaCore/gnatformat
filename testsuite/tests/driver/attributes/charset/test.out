--  test.adb
with Test_Dep;

procedure Test is
begin
   null;
end Test;

--  test_dep.ads
package Test_Dep is
   Indicator_Nice : constant array (Indicator_Range) of Wide_Wide_Character :=
     ('◴', '◷', '◶', '◵');
end Test_Dep;

